hooks/build

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker build --build-arg OS=linux  --build-arg ARCH=amd64 -f $DOCKERFILE_PATH -t $IMAGE_NAME .
```

Comment:
What is this for?

Reply:
not need. I will delete it. Cuz I setup my fork to build via dockerhub. This is a hook for dockerhub automated build.